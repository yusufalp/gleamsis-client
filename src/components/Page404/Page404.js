import React from 'react';

class Page404 extends React.Component {
  render() {
    return (
      <div>
        <h1>Page not found</h1>
      </div>
    )
  }
}

export default Page404;