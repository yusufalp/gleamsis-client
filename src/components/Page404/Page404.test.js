import React from 'react';
import ReactDOM from 'react-dom';
import Page404 from './Page404';
import renderer from 'react-test-renderer';

describe('Page404 component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Page404 />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders the UI as expected', () => {
    const tree = renderer
      .create(<Page404 />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
})