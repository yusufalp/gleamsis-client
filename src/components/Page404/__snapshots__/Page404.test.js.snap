// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page404 component renders the UI as expected 1`] = `
<div>
  <h1>
    Page not found
  </h1>
</div>
`;
